<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="main_frame">
     <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#main_frame{
background-color: rgb(255,255,255);
boarder-color: rgb(0,0,0);
border-radius: 15px;
border: 5px dotted '#95C9F5';
}
</string>
   </property>
   <widget class="QProgressBar" name="pg_bar_name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QToolButton" name="fileUpload">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#fileUpload{
boarder: 0.5px;
background-color:'#97A8DE';
border-radius: 15px;
}


</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#FFFFFF';
border-radius: 15px;
border: 5px solid '#A5907E';
color: '#A5907E';</string>
    </property>
    <property name="text">
     <string>Add your data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>580</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dfdf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#FFFFFF';
border-radius: 15px;
border: 5px solid '#B2EBF4';
color: '#AAC6FA';

</string>
    </property>
    <property name="text">
     <string>Exit Program</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="excel_edit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>540</width>
      <height>290</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <zorder>pg_bar_name</zorder>
   <zorder>start_btn</zorder>
   <zorder>exit_btn</zorder>
   <zorder>excel_edit</zorder>
   <zorder>fileUpload</zorder>
   <zorder>logo_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
